src/test/java/com/amazon/ion/impl/TextWriterTest.java

```diff
@@ -616,6 +618,11 @@ public void testAnnotationNotSetToIvmOnStartOfStream()
         super.testAnnotationNotSetToIvmOnStartOfStream();
     }
 
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
We should probably add 1 or 2 test cases to ensure that the max length is respected when changed.

Reply:
See https://github.com/amazon-ion/ion-java/pull/784/files#diff-4c5ab165f869b42ee1dafea60645ea4ca712608ea0254f13b93dfda74e4ea617R3177

Or did you mean something else?

Reply:
That's exactly it. I guess I missed that because I was expecting the TextWriter tests to be in `TextWriterTest.java`.